atd-vze/src/views/Crashes/crashGridTableParameters.js

```diff
@@ -244,7 +244,7 @@ export const crashGridTableAdvancedFilters = {
       },
       {
         id: "geo_afd",
-        label: "Remove Austin Full Purpose",
+        label: "Include Outside Of Austin Full Purpose",
```

Comment:
I don't think that the VZ team will have any confusion here now! Super small detail, but I noticed that there is inconsistent capitalization on one toggle label - `Has been Geocoded`. Not a big deal but wondering if we should patch that one up while you are in here.

Reply:
I think this is a great idea. Polish to the UI is never a bad thing!

Reply:
thanks for catching that mike, just fixed that capitalization inconsistency!